### PR TITLE
TEVA-808 Search no results ux

### PR DESCRIPTION
**Pull request by alexbowen**
## Jira ticket URL

https://dfedigital.atlassian.net/browse/TEVA-808

## Changes in this PR:

- a couple of minor improvements also in this PR

## Screenshots of UI changes:

Screenshot 2020-06-19 at 11 31 53

### CODE REVIEW

File: app/frontend/src/search/ui/input/radius.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { getRadius } from './radius';
+
+describe('getRadius', () => {
+  test('to be true when radius data attribute is present', () => {
```
Comment (cpjmcquillan) [inline app/frontend/src/search/ui/input/radius.test.js:4]: maybe `to be truthy` instead?

File: app/views/vacancies/_filters.html.haml [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
     .govuk-form-group.govuk-inset-text#location-radius-select
       = label_tag 'radius', t('jobs.filters.within_radius'), class: 'govuk-label'
-      = select_tag 'radius', options_for_select(radius_filter_options, @vacancies_search.radius), disabled: @vacancies_search.disable_radius?, class: 'govuk-select govuk-!-width-full'
+      = select_tag 'radius', options_for_select(radius_filter_options, @vacancies_search.radius), disabled: @vacancies_search.disable_radius?, class: 'govuk-select govuk-!-width-full', 'data-radius': 10
```
Comment (cpjmcquillan) [inline app/views/vacancies/_filters.html.haml:14]: In rails you can do `{ data: { radius: 10 } }` for the same effect I _think_, but if this works it's fine with me

File: app/views/vacancies/index.html.haml [hunk 3]
```diff
@@ -23,10 +23,18 @@
             = render partial: 'vacancy', locals: { vacancy: vacancy }
 
       - elsif @vacancies.user_search?
+        %div.divider-bottom
```
Comment (cpjmcquillan) [inline app/views/vacancies/index.html.haml:26]: You can just do `.divider-bottom`

File: app/views/vacancies/_filters.html.haml (hunk 2 above)
Comment (alexbowen) [reply app/views/vacancies/_filters.html.haml]: your suggestion is neater

**Pull request merged**